cellprofiler/modules/trackobjects.py

```diff
@@ -330,6 +338,18 @@ def create_settings(self):
             specified <b>Measure</b> module previous to this module in the pipeline so
             that the measurement values can be used to track the objects.</li>
 
+            <li><i>%(TM_FOLLOWNEIGHBORS)s:</i> Uses the multiobject tracking approach described by
+            <i> Delgado-Gonzalo et al., 2010</i>. This approch is assuming that objects move in coordinated
+            way (contrary to LAP). An object movement direction is more likely to be in the agreement with 
```

Comment:
> An object movement direction is more likely to be in the agreement with the movement directions of its "neighbors".

An object's movement direction is more likely to be in agreement with the movement directions of its "neighbors".